core/menu-structure.json

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
Note to self, this should have some header comment that says it's auto generated.

Reply:
Json files don't allow comments, like yarn.lock does. nevermind.